cmd/libs/go2idl/client-gen/generators/tags.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators
+
+import (
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/cmd/libs/go2idl/types"
+)
+
+// extractBoolTagOrDie gets the comment-tags for the key and asserts that, if
+// it exists, the value is boolean.  If the tag did not exist, it returns
+// false.
+func extractBoolTagOrDie(key string, lines []string) bool {
```

Comment:
Just 2 plus the fake, so 3.  The API we want here is just not "pretty" for a common lib, any yet it is duplicated in a few pkgs.  This is sort of common to Go, where "a little duplication is better than a little dependency".  I think that the other users of this will be eliminated as their codegen is cleaned up, too, so I think we should leave it as-is for now.  If you feel strongly, I can just move the OrDie func into the common lib.